src/main/protobuf/subra/Robot.proto

```diff
@@ -0,0 +1,50 @@
+syntax = "proto3";
+option java_package = "io.leonis.subra.protocol";
+
+message Command {
+  message Move {
+    float x = 1;
+    float y = 2;
+    float r = 3;
+  }
+
+  message Action {
+   float dribble = 1;
+   float kick    = 2;
+   float chip    = 3;
+  }
+
+  message Setting {
+    uint32 id    = 1;
+    string name  = 2;
+    string value = 3;
+  }
+
+           uint32  id       = 1;
+           Move    move     = 2;
+           Action  action   = 3;
+  repeated Setting settings = 4;
+}
+
+message Measurements {
+  message Single {
+    string label             = 1;
+    float  value             = 2;
+    int32  tenFoldMultiplier = 3;
+  }
+           uint32 id           = 1;
```

Comment:
I think we should rename this field to `robot_id` or `sequence_id` depending on what it actually is.